File: CS/Form1.cs
using System;

using System.Windows.Forms;

namespace CustomTodayExample {
    public partial class Form1 :Form {
        public Form1() {
            InitializeComponent();
            dateEdit1.Properties.TodayDate = new System.DateTime(2015, 11, 30, 0, 0, 0, 0);
        }
    }
}

File: VB/Form1.vb
Imports System

Imports System.Windows.Forms

Namespace CustomTodayExample
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			dateEdit1.Properties.TodayDate = New Date(2015, 11, 30, 0, 0, 0, 0)
		End Sub
	End Class
End Namespace